fashion-messaging-app-ui/src/RequireLogin.jsx

```diff
@@ -0,0 +1,14 @@
+import { Navigate, useLocation } from 'react-router-dom';
```

Comment:
Navigate helps me change current routes, useLocation checks what location/route my app is in